exercises/all-your-base/all_your_base_test.rb

```diff
@@ -175,7 +179,7 @@ def test_negative_digit
     digits = [1, -1, 1, 0, 1, 0]
     input_base = 2
     output_base = 10
-    expected = nil
+    expected = WHAT TO DO?
```

Comment:
Currently we just expect the output to be nil. Should the cases with invalid data be changed to expect an error to be raised?